forward/godefs.lua

```diff
@@ -20,9 +20,10 @@ function M.Init()
    return golib.Init()
 end
 
-function M.AddBackend(service, ip, ip_len)
+function M.AddBackend(service, ip, ip_len, health_check_type)
    return golib.AddBackend(GoString(service, #service),
-                           GoSlice(ip, ip_len, ip_len))
+                           GoSlice(ip, ip_len, ip_len),
+						   health_check_type)
```

Comment:
Needs gofmt

Reply:
Oops, yeah it wasn't caught by gofmt since it's not a Go file. Also completely unrelatedly, there's actually existing inconsistent indentation in that file at lines 4 - 7 (four spaces instead of three).